storm-server/src/main/java/org/apache/storm/scheduler/resource/ResourceAwareScheduler.java

```diff
@@ -226,7 +226,17 @@ private void scheduleTopology(TopologyDetails td, Cluster cluster, final User to
                         //Only place we fall though to do the loop over again...
                     } else { //Any other failure result
                         //The assumption is that the strategy set the status...
-                        topologySubmitter.markTopoUnsuccess(td, cluster);
+                        String msg = "";
```

Comment:
This can be replaced by result.toString()

Reply:
Changed